safekeeper/client/src/mgmt_api.rs

```diff
@@ -8,7 +8,7 @@ use std::error::Error as _;
 use http_utils::error::HttpErrorBody;
 use reqwest::{IntoUrl, Method, StatusCode};
 use safekeeper_api::models::{
-    PullTimelineRequest, PullTimelineResponse, SafekeeperUtilization, TimelineCreateRequest,
+    self, PullTimelineRequest, PullTimelineResponse, SafekeeperUtilization, TimelineCreateRequest,
```

Comment:
nit: all other `*Request`/`*Response` structs are used explicitly (without `models::`). Maybe it makes sense to do the same for `TimelineMembershipSwitchRequest`/`TimelineDeleteResult`/etc for consistency

Reply:
thanks for raising. I'd like us to get rid of those direct uses as they cause merge conflicts and the diff is not very useful either. that's why I did it in this PR as a first step.